contrib/friendz/src/friendz.nit

```diff
@@ -972,6 +957,25 @@ redef class Game
 	# ResizeButton
 	var button_size = new ResizeButton(self)
 
+	init
+	do
+		load_levels
+		init_buttons
+		entities.clear
+		title
+	end
+
+	#loading sounds, must be redef by specific platforms
```

Comment:
Is this supposed to be a synopsys? It's not looking like a synopsys.

Add a space after the `#` and change the first letter for an uppercase.

Also, maybe review the comment to explain that all specific sounds for the game are loaded here.